platform/nvidia-bluefield/installer/create_sonic_image

```diff
@@ -0,0 +1,420 @@
+#!/bin/bash
+#
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -x
+set -e
+
+WDIR=
+IMAGE_VERSION=
+KERNEL_VERSION=
+CHROOT_DIR=
+INITRD=
+VMLINUZ=
+MODULES_DIR=
+
+SIGNING_CERT=
+SIGNING_KEY=
+CDIR=/sonic/
+SDIR=$CDIR/platform/${CONFIGURED_PLATFORM}/installer/
+DDIR=$CDIR
+CAPSULE=/lib/firmware/mellanox/boot/capsule/boot_update2.cap
+BOOTCTL_DRIVER=
+TMFIFO_DRIVER=
+SDHCI_OF_DWCMSHC_DRIVER=
+TARGET_MACHINE=nvidia-bluefield
+BFB="${BFB:-/lib/firmware/mellanox/boot/default.bfb}"
+GRUB_AA64=grubnetaa64.efi
+GRUB_CFG="" # Common Grub Config
+BF2_BOOT_ARGS="console=ttyAMA1 console=hvc0 console=ttyAMA0 earlycon=pl011,0x01000000 earlycon=pl011,0x01800000"
+BF2_GRUB_CFG="$BF2_BOOT_ARGS isolcpus=1-7 nohz_full=1-7 rcu_nocbs=1-7"
```

Comment:
we don't need it anymore. But I don't want to change this file right now, because this will require additional testing and we don't have time for this. I suggest to submit it as is with BF2 support and cleanup it later